app/presenters/publishing_api/policy_group_presenter.rb

```diff
@@ -49,12 +49,7 @@ def details
     end
 
     def body
-      # It looks 'wrong' using the description as the body, but it isn't
-      if item.description.present?
-        Whitehall::GovspeakRenderer.new.govspeak_with_attachments_to_html(item.description, item.attachments, item.email)
-      else
-        ""
```

Comment:
Is this empty string important in the frontend app? I.e., will changing this from "" to nil break something?

Reply:
I didn't read your commit comment 🤦. Ignore this